infrastructures/openshift/src/test/java/org/eclipse/che/workspace/infrastructure/openshift/multiuser/oauth/OpenshiftTokenInitializationFilterTest.java

```diff
@@ -128,26 +124,27 @@ public void extractSubjectCreatesSubjectWithCurrentlyAuthenticatedUser()
     assertEquals(subject.getUserName(), USERNAME);
   }
 
-  @Test
-  public void handleMissingTokenShouldAllowUnauthorizedEndpoint()
-      throws ServletException, IOException {
-    when(servletRequest.getServletPath()).thenReturn("/system/state");
-
-    openshiftTokenInitializationFilter.handleMissingToken(
-        servletRequest, servletResponse, filterChain);
-
-    verify(filterChain).doFilter(servletRequest, servletResponse);
-  }
-
-  @Test
-  public void handleMissingTokenShouldRejectRequest() throws ServletException, IOException {
-    when(servletRequest.getServletPath()).thenReturn("blabol");
-
-    openshiftTokenInitializationFilter.handleMissingToken(
-        servletRequest, servletResponse, filterChain);
-
-    verify(servletResponse).sendError(eq(401), anyString());
-  }
+  // TODO: move to abstract class test
```

Comment:
do you plan to do this in this PR or to create an issue for that?

Reply:
thanks, I'll update the PR. I've overlooked this

Reply:
fixed